Donut/templates/donut.html

```diff
@@ -0,0 +1,24 @@
+{% extends "layout.html" %}
+{% block body %}
+
+<div class="container theme-showcase" role="main">
+
+    <!-- Main jumbotron for a primary marketing message or call to action -->
+    <div class="jumbotron">
+        <img src="static/images/newdonut.png" alt="d(O)nut" title="d(O)nut" style="margin: auto; display: block">
```

Comment:
The `url_for('static', filename='images/newdonut.png')` syntax is generally better. For static files it's probably fine to hard code the url, but for other endpoints you definitely want to use the `url_for` function. It helps a lot if you decide to redo how the APIs are laid out.